src/ui/public/styles/variables/depth.less

```diff
@@ -0,0 +1,5 @@
+/**
+ * 1. The local nav contains tooltips which should pop over the filter bar.
+ */
+@globalDepthFilterBar: 2; /* 1 */
+@globalDepthLocalNav: 3; /* 1 */
```

Comment:
@cjcenizal I'm a bit confused by the term "global" here. z-index isn't a global setting, it depends on the local [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context). I'm a little worried these variable names and their location in a single global file could mislead someone about their relationship. For instance, given two variables with the values 40 and 2 in this file, depending on the stacking context of the elements they're applied to the 40 might actually appear below the 2.